api/src/opentrons/hardware_control/ot3api.py

```diff
@@ -1913,41 +1928,54 @@ async def blow_out(
             blowout_spec.instr.set_current_volume(0)
             blowout_spec.instr.ready_to_aspirate = False
 
-    async def get_tip_presence_status(self, mount: OT3Mount) -> bool:
-        """
-        Check tip presence status. If a high throughput pipette is present,
-        move the tip motors down before checking the sensor status.
-        """
-        checked_mount = OT3Mount.from_mount(mount)
-        high_throughput = self._gantry_load == GantryLoad.HIGH_THROUGHPUT
-        if high_throughput:
-            # check if we're already at the tip_presence_check position, if we are dont move
+    @contextlib.asynccontextmanager
+    async def _high_throughput_tip_action(self) -> AsyncIterator[None]:
```

Comment:
```suggestion
    async def _high_throughput_check_tipsself) -> AsyncIterator[None]:
```
maybe? `tip_action` is sort of nonspecific